### ISSUE TITLE
App Crash in simulator or device

### ISSUE THREAD

**Issue by maresp85** [expo/expo#21745]
### Summary

Hi! few days ago with the compiled apps dont have any problem with the apk in devices or simulator, but from yesterday appear this error:

**Error**: Exception in HostObject::get for prop 'ExpoMediaLibrary': expo.modules.kotlin.exception.MissingTypeConverter: Cannot find type converter for 'expo.modules.kotlin.Promise'., js engine: hermes
2023-03-16 21:35:14.348 22896-22944 ReactNativeJS           pid-22896                            I  Running "main
2023-03-16 21:35:14.348 22896-22944 ReactNativeJS           pid-22896                            E  Invariant Violation: "main" has not been registered. This can happen if:
                                                                                                    * Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
                                                                                                    * A module failed to load due to an error and `AppRegistry.registerComponent` wasn't called., js engine: hermes
2023-03-16 21:35:14.350 22896-22945 AndroidRuntime          pid-22896                            E  FATAL EXCEPTION: mqt_native_modules                                                                                                   
                                                                                                    com.facebook.react.common.JavascriptException: Error: Exception in HostObject::get for prop 'ExpoMediaLibrary': expo.modules.kotlin.exception.MissingTypeConverter: Cannot find type converter for 'expo.modules.kotlin.Promise'., js engine: hermes, stack:
                                                                                                    requireNativeModule@1:614428
                                                                                                    anonymous@1:1372735
                                                                                                    loadModuleImplementation@1:40022
                                                                                                    guardedLoadModule@1:39571
                                                                                                    metroRequire@1:39199
                                                                                                    anonymous@1:1366655
                                                                                                    loadModuleImplementation@1:40022
                                                                                                    guardedLoadModule@1:39571
                                                                                                    metroRequire@1:39199
                                                                                                    anonymous@1:1310017
                                                                                                    loadModuleImplementation@1:40022


In this moment i can't deploy my apk in the devices. Thank you!

### What platform(s) does this occur on?

Android

### SDK Version

48

### Environment

  "@react-native-async-storage/async-storage": "^1.17.11",
  "@react-native-community/datetimepicker": "6.7.3",
  "@react-native-community/netinfo": "^9.3.7",
  "@react-native-picker/picker": "^2.4.8",
  "@react-navigation/bottom-tabs": "^6.5.7",
  "@react-navigation/native": "^6.1.6",
  "@react-navigation/stack": "^6.3.16",
  "@rneui/base": "^4.0.0-rc.7",
  "@rneui/themed": "^4.0.0-rc.7",
  "axios": "^0.24.0",
  "expo": "~48.0.4",
  "expo-camera": "~13.2.1",
  "expo-device": "~5.2.1",
  "expo-file-system": "~15.2.2",
  "expo-image-picker": "~14.1.1",
  "expo-linking": "~4.0.1",
  "expo-location": "~15.1.1",
  "expo-media-library": "^15.2.2",
  "expo-notifications": "~0.18.1",
  "expo-status-bar": "~1.4.4",
  "expo-updates": "~0.16.3",
  "moment": "^2.29.4",
  "npm": "^9.6.0",
  "react": "18.2.0",
  "react-native": "0.71.3",
  "react-native-flash-message": "^0.4.0",
  "react-native-gesture-handler": "^2.9.0",
  "react-native-safe-area-context": "^4.5.0"


### Minimal reproducible example

![2023-03-17_07-45](https://user-images.githubusercontent.com/11510312/225908277-69f04ce3-d9f8-4ce3-954a-62ed04586914.png)

**Comment by lc3t35** [comment]
[SDK48, Android] I also have the same error (after fixing #21673, #21477)

**Comment by jonjamz** [comment]
I am getting this same error on SDK48, Android. Any kind of usage of MediaLibrary causes it to crash on start.

**Comment by DayUx** [comment]
I have the same issue

**Comment by lc3t35** [comment]
Solved by #21711 by adding in package.json (see

```
"resolutions": {
    "expo-modules-core": "1.2.4",
    "@expo/config-plugins": "5.0.4"
  },
```

**Comment by maresp85** [comment]
> Solved thanks to #21711 by adding in package.json
> 
> ```
> "resolutions": {
>     "expo-modules-core": "1.2.4",
>     "@expo/config-plugins": "5.0.4"
>   },
> ```
> 
> and update is available now : expo-media-library ~15.2.2 → ~15.2.3

With the expo-media-library update, the apk works! thanks!
